src/client-functions/selector-builder/index.js

```diff
@@ -144,6 +146,22 @@ export default class SelectorBuilder extends ClientFunctionBuilder {
         if (snapshot && !this.options.counterMode) {
             this._addBoundArgsSelectorGetter(snapshot, selectorArgs);
             createSnapshotMethods(snapshot);
+
+            if (this.options.customMethods) {
```

Comment:
I didn't get it, what's the difference between `snapshot.hasClass()` and `snapshot.hasMyCustomProperty()`?